### PR TITLE
Action Reference - PR #2

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

More work on the Actions Reference pages.

This one took ages because while it covers very few pages, one of those is the landing page. 

So we have here:

* I divided the landing page by category, and organised by the UI in each category. I think it's easier to scan. I did not, however, change the TOC - that's still alphabatised. Would appreciate your opinion on this, @JonMUnity .
* Also, note there are comments on that page. Feel very free to answer :)
* Rewrote Center Pivot.
* Combined three Set Pivot To Selection pages into one. There wasn't any real point in making them three pages, as there wasn't much interesting info to distinguish them. I used the nicest of the three diagrams as a single example.
* Some small edits to pages from PR #1.

Spoiler alert: The next few PRs will be based on the new order of the pages in the landing page. 

### Links

**Jira:** DOCATT-5849

### Comments to Reviewers

See the Purpose of this PR section.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=560) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=560) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: Sue Arkin<br/>:x: suearkinunity<br/><hr/>**Sue Arkin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=560) it.</sub>

File: Documentation~/CenterPivot.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-The __Center Pivot__ action moves the pivot point for the Mesh to the center of the object’s bounds. For example, if you move a lot of vertices on one side of your Mesh, when you try to rotate the object, it rotates around a point outside of the Mesh. 
+The __Center Pivot__ action moves the pivot point for a mesh to the center of the object’s bounds. You might want to do this if you have elongated the vertices on one side of your mesh, which pushes the bounds on that side far from the original center.
 
-From the main menu, select **Tools** > **ProBuilder** > **Object** > **Center Pivot**.
+To center the pivot point:
 
-![Center Pivot example](images/CenterPivot_Example.png)
+1. Select one or more objects.
+    > **Note:** If you have multiple objects selected, each object's new pivot point is the center of that object, regardless of the position of any other object. 
+1. From the main menu, select **Tools** > **ProBuilder** > **Object** > **Center Pivot**.
```
Comment (suearkinunity) [inline Documentation~/CenterPivot.md:9]:
```suggestion
1. From the main menu, select **Tools** > **ProBuilder** > **Object** > **Center Pivot**.
    If you are in the GameObject context, you can also right-click the object and select **ProBuilder** > **Center Pivot**.
```

File: Documentation~/Toggle_X_Ray.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Toggle X Ray 
+
+Shows hidden faces.
+
+<!--
+Note to reviewers: this is a placeholder; the page itself will be written in a different PR. 
+
+[[From the what's new page: * Added a new [Selection X-Ray](preferences.md#sel-xray) option to highlight hidden element selections with a muted color. The default shortcut is **Ctrl/Alt+Shift+X** (modifiable in the Shortcuts Manager), and you can also access it through the ProBuilder menu (**Tools** > **ProBuilder** > **Interaction** > **Toggle X Ray**).]]
+
+[[why is this both a preference an an action?]]-->
```
Comment (JonMUnity) [inline Documentation~/Toggle_X_Ray.md:10]: I don't really know much about this feature. @modrimkus, can you explain this one?

File: Documentation~/ref_actions.md [hunk 3]
```diff
@@ -1,17 +1,74 @@
...
+
+These actions are available:
+
+* From the **main menu** > **Tools** > **ProBuilder**. The actions are grouped in **Selection**, **Interaction**, **Object**, and **Geometry**.
+* From the context menu: 
+    1. In the **Tools** overlay, set the working context to ProBuilder. 
+    1. In the **Tool Settings** overlay, select an edit mode.
+    1. In the **Scene** view, right-click on a ProBuilder object.
+
+<!--So... I divided this page by category. I think it's easier to scan. But then the question is: should the TOC match, or should the TOC stay alphabatised? -->
```
Comment (JonMUnity) [inline Documentation~/ref_actions.md:13]: Good question. I think we should make the TOC match. If a user is going through it alphabetically, then the implication is that they know what they're looking for. If they know what they're looking for, why wouldn't they use CTRL+F?

File: Documentation~/ref_actions.md [hunk 4]
```diff
@@ -1,17 +1,74 @@
...
+
+## Selection actions
+
+These options are never available in the GameObject tool context.
```
Comment (JonMUnity) [inline Documentation~/ref_actions.md:17]:
```suggestion
These options are not available in the GameObject tool context.
```

File: Documentation~/ref_actions.md [hunk 5]
```diff
@@ -1,17 +1,74 @@
...
+| [Select Hole](Selection_SelectHole.md) | x | x | |
+| [Select Loop (in Edge Selection mode)](Selection_Loop_Edge.md) | | x | |
+| [Select Loop (in Face Selection mode)](Selection_Loop_Face.md) | | | x |
+| [Select Material](Selection_SelectByMaterial.md) | | | x |
+| [Select Ring (in Edge Selection mode)](Selection_Ring_Edge.md) | | x | |
+| [Select Ring (in Face Selection mode)](Selection_Ring_Face.md) | | | x |
+| [Select Vertex Color](Selection_SelectByVertexColor.md) | x | x | x |
+| [Shrink Selection](Selection_Shrink.md) | | x | x | x |
+
+<!--two select loop should be one, two select ring should be one - that will have to be another ticket-->
```
Comment (JonMUnity) [inline Documentation~/ref_actions.md:31]: Don't forget to delete this and other comments.

File: Documentation~/ref_actions.md [hunk 6]
```diff
@@ -1,17 +1,74 @@
...
+
+* From the **main menu** > **Tools** > **ProBuilder**. The actions are grouped in **Selection**, **Interaction**, **Object**, and **Geometry**.
+* From the context menu: 
+    1. In the **Tools** overlay, set the working context to ProBuilder. 
+    1. In the **Tool Settings** overlay, select an edit mode.
+    1. In the **Scene** view, right-click on a ProBuilder object.
+
+<!--So... I divided this page by category. I think it's easier to scan. But then the question is: should the TOC match, or should the TOC stay alphabatised? -->
+
+## Selection actions
```
Comment (JonMUnity) [inline Documentation~/ref_actions.md:15]: The categories are a big improvement, I think.

File: Documentation~/ref_actions.md (hunk 3 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: Oh man was that old news :)

File: Documentation~/ref_actions.md (hunk 4 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: Ignoring this since other PRs killed this page anyway

File: Documentation~/ref_actions.md (hunk 5 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: Ignoring this since other PRs killed this page anyway

File: Documentation~/ref_actions.md (hunk 6 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: I know, I'm a genius.

File: Documentation~/ref_actions.md (hunk 4 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: Oh, well - a future PR in a branch I'm already working on

File: Documentation~/ref_actions.md (hunk 5 above)
Comment (suearkinunity) [reply Documentation~/ref_actions.md]: Oh, well - a future PR in a branch I'm already working on

File: Documentation~/TableOfContents.md [hunk 7]
```diff
@@ -109,8 +111,8 @@
 * [ProBuilder Preferences](preferences)
 * [Troubleshooting](faq)
 * [Experimental features](experimental)
-	* [Create and edit Bezier shapes](workflow-create-bezier)
-	* [Perform Boolean operations](boolean)
-	* [Store meshes as assets](meshes-as-assets)
+	* [Bezier Shape tool](workflow-create-bezier)
```
Comment (suearkinunity) [inline Documentation~/TableOfContents.md:114]: I have a feeling I don't want this change. Please don't merge until I've figured it out.

File: Documentation~/Toggle_X_Ray.md (hunk 2 above)
Comment (modrimkus-unity) [reply Documentation~/Toggle_X_Ray.md:10]: @suearkinunity It toggles if selected elements that are occluded by geometry are still visualized.

https://github.com/user-attachments/assets/927cd67f-e3a8-42a5-832b-cd6fb5b22d52

For why it's a pref but also has a shortcut - I'm not sure. @gabriel-unity thoughts? : )

Comment (modrimkus-unity) [reply Documentation~/Toggle_X_Ray.md:10]: @gabrielw-us thoughts? : )

**Comment by suearkinunity** [comment]
Merging into my working branch (so not into master - nobody panic).

**Pull request merged**